Source/Core/Core/HotkeyManager.cpp

```diff
@@ -69,6 +69,9 @@ const std::string hotkey_labels[] = {
     _trans("Connect Wii Remote 4"),
     _trans("Connect Balance Board"),
 
+    _trans("Next Profile"),
```

Comment:
These should probably be `"Next Controller Profile"` and `"Previous Controller Profile"` to disambiguate.

Reply:
Fixed.